Updated dummy classes to not inherit from Document

orangejenny opened the pull request:
Minor, just wanted to stop these classes from cluttering up metrics on how many couch-based classes are left to migrate to sql.

These are quite old: https://github.com/dimagi/commcare-hq/commit/8d063f6737b03aabab8ef4f64edf811f68510f49

I did verify locally that running `./manage.py test corehq.apps.hqcase` still finds tests.

orangejenny committed:
Updated dummy classes to not inherit from Document

millerdev commented on corehq/apps/hqcase/models.py:
```diff
@@ -1,6 +1,5 @@
-# This file is only here so that django will recognize that 
+# This file is only here so that django will recognize that
 # this is a valid app and run the associated unit tests.
```
If this comment is still true then I think this file can be removed since we now use nose and nose will find tests anywhere, not just in django apps.

dannyroberts commented:
Nice find @orangejenny! @millerdev I think you're onto something, but alas that comment existed in the file before my 2011 (pre-PR era!) commit that Jenny referenced.

@orangejenny at the time the important thing definitely was subclassing `Document`, not just having any old class defined in models.py. At the time I believe it was to get couchdbkit to recognize the django app as relevant to couchdbkit—perhaps without it it wouldn't sync views defined in the app, or something like that. So i think that's what I'd test, and if it works, I'd just keep a blank file here (or no file?) because in that case I don't think `class _(object): pass` is doing anything

orangejenny committed:
Removed cleanup and crs_reports models files

orangejenny committed:
Reverted fluff changes

orangejenny committed:
Removed hqcase models file

orangejenny commented:
Very interesting, thanks for the context. The non-fluff files are in apps that no longer have any couch views, so for those I deleted the `_` classes and model files altogether. For fluff, I just reverted my changes, rather than dig into whether or not these dummy classes are still needed.
